apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-welcome-tour/src/tour-launch.js

```diff
@@ -18,8 +18,10 @@ import { Icon } from '@wordpress/icons';
 import { useDispatch, useSelect } from '@wordpress/data';
 import { createPortal, useEffect, useState } from '@wordpress/element';
 import { registerPlugin } from '@wordpress/plugins';
+import { recordTracksEvent } from '@automattic/calypso-analytics';
 
 function LaunchWpcomWelcomeTour() {
+	const [ portalParent ] = useState( () => document.createElement( 'div' ) );
```

Comment:
DOM elements aren't usually placed in "state". But I see that you don't want a new one created each render. Do you want `useRef` maybe instead of `useState`?

Reply:
Ah, thank you!!  I just went to do read up on my understanding of that and found [this](https://www.codebeast.dev/usestate-vs-useref-re-render-or-not/) super helpful:

1. useState causes re-render; useRef does not.
2. Both useState and useRef remembers their data after a re-render